code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -226,10 +236,75 @@
 	else
 		return
 
+/mob/living/simple_animal/bot/medbot/proc/tip_over(mob/user)
+	playsound(src, 'sound/machines/warning-buzzer.ogg', 50)
+	user.visible_message("<span class='danger'>[user] tips over [src]!</span>", "<span class='danger'>You tip [src] over!</span>")
+	mode = BOT_TIPPED
+	var/matrix/mat = transform
+	transform = mat.Turn(180)

```

Comment:
would this look better if the medibot was only rotated by 90 degrees?

Reply:
yes it looks a bit better if it's 90 degrees